requirements.txt

```diff
@@ -1,18 +1,18 @@
-h5py==2.10.0
+h5py>=2.10.0,<3
 jupyter>=1.0.0,<2
-jupyter_contrib_nbextensions==0.5.1
-matplotlib==2.2.2
+jupyter_contrib_nbextensions>=0.5.1,<1
+matplotlib>=2.2.2,<3
 git+git://github.com/ionpath/mibilib@v1.2.6
-netCDF4==1.5.3
-numpy==1.16.3
-pandas==0.24.2
-scikit-image==0.16.2
-scikit-learn==0.21.2
-scipy==1.2.1
-seaborn==0.10.1
-statsmodels==0.11.1
-tables==3.6.1
-umap-learn==0.4.6
-xarray==0.12.3
+netCDF4>=1.5.3,<2
+numpy>=1.16.3,<2
+pandas>=0.23.3,<1
+scikit-image>=0.14.1,<=0.16.2
```

Comment:
should we do `<1` here instead of `<=0.16.2`?

Reply:
Let me try it out. I assumed that `scikit-image` would break on our repo with more recent versions but didn't actually test it.

Reply:
Yeah, confirmed that it won't work for anything greater than `0.16`. Problem is `skimage` discontinued support for `external` after `0.16`, but some of our code still relies on that.